assets/js/components/Message/index.test.js

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react'
+import { shallow } from 'enzyme'
+import Message from './index'
+
+describe('Message', () => {
+    it('renders message component', () => {
+        const wrapper = shallow(<Message />);
```

Comment:
This is really testing out that enzyme shallow mounts it, right? This isn't really testing the presence of anything within the message

Reply:
I added tests for receiving props and displaying a message.